group-aws/bestpractices.md

```diff
@@ -0,0 +1,19 @@
+# Best practices
```

Comment:
I guess it's more "these are the specific things we care about working towards fixing" and in future "these are the modules that need fixing" and "these are the PRs that will address those". I don't mind being overly documented.